app/javascript/components/admin/mission/Form.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+
+const Hidden = styled.div`
+  display: none;
+`
+
+const Form = ({ piercedLocation }) => {
+  const keys = Object.keys(piercedLocation)
+  const loc = piercedLocation
+  console.log(loc)
+  return (
+    <Hidden>
+      {keys.map(key =>
+        loc[key].map((lines, idx) => (
+          <select
+            multiple
+            key={Math.random()}
+            name={`locations[${key}][${idx}][]`}
+            defaultValue={lines}>
+            {_.map(lines, line => <option key={Math.random()} value={line} />)}
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'_' is not defined. (no-undef)